pyBeam/src/beam.cpp

```diff
@@ -203,57 +188,58 @@ void CBeamSolver::Solve(void){
 		std::cout << "#####    EXITING ITERATIVE SEQUENCE   #####" << std::endl;
 	}
   
-  //addouble pos1, pos2, pos3;
-  //addouble grad_t;
-  
-  //pos1 = structure->GetDisplacement(100, 0);
-  //pos2 = structure->GetDisplacement(100, 1);
-  //pos3 = structure->GetDisplacement(100, 2);
-  
-  //cout << pos1 << " " << pos2 << " " << pos3 << endl;
-  
-  //globalTape.registerOutput(pos2);
-  
-  //globalTape.setPassive();
-  
-  //pos2.setGradient(1.0);
-  
-  //globalTape.evaluate();
-  
-  //grad_t = thickness.getGradient();
-   
-  //std::cout << " t' is " << grad_t << endl;
-
 }
 
-passivedouble CBeamSolver::ExtractDisplacements(int iNode, int iDim){
+passivedouble CBeamSolver::OF_NodeDisplacement(int iNode){
```

Comment:
This is the objective function, computes the norm of the displacement of node iNode.